arcus/azureml/projectutils.py

```diff
@@ -2,6 +2,7 @@
 The projectutils module provides common operations related to data/ai projects
 '''
 import os
+from azureml.core import Workspace
```

Comment:
We should exclude this file? Its overwriting the latest changes?

Reply:
Well, the workspace logic has been moved to the environment (where everything is abstracted away in an 'interface' based implementation that returns a local environment or an azureml environment

Regarding projectutils: it could make sense to move that to the workenvironment as well, indeed, but would do that out of this PR